### ISSUE TITLE
fix: selection range remains visible when an option is selected in combobox 

### ISSUE THREAD

**Issue by simonxabris** [microsoft/fast#5833]
# 🐛 Bug Report

When the combobox is in `inline` or `both` mode and you start typing and use the cursor to select an option, the option gets selected but the inline selection remains visible in the input. I think this is not the desired behaviour and the selection should be cleared. It is already implemented when the option is selected using the `Enter` key.

## 💻 Repro or Code Sample

You can reproduce it in the [explorer](https://explore.fast.design). Set the combobox to `autocomplete="both"`, start typing and then select an option via the mouse. You will see that the selection stays visible.

<details>
<summary>Repro video</summary>

https://user-images.githubusercontent.com/27497229/163234563-d62cbe7f-bbff-4949-ad72-e511292c2c74.mov

</details>

## 🤔 Expected Behavior

The selection is cleared.

## 😯 Current Behavior

The selection remains visible.

## 💁 Possible Solution

It's probably a 1-2 line fix as it's already implemented for `Enter` selection. I'll submit a PR about this.

**Comment by radium-v** [comment]
Fixed by #5834
